features/steps/steps.py

```diff
@@ -165,17 +165,16 @@ def step_impl(context, attribute, value):
     )
 
 
-@given('A file with {field} "{value}"')
-def step_impl(context, field, value):
+@given('A file with {field} "{values}"')
+def step_impl(context, field, values):
+    values = list(values.replace('"', "").split(', '))
     if field == "Model View Definition":
-        applicable = get_mvd(context.model) == value
+        applicable = any([get_mvd(context.model) == value for value in values])
     elif field == "Schema Identifier":
-        applicable = context.model.schema.lower() == value.lower()
+        applicable = any([context.model.schema.lower() == value.lower() for value in values])
     else:
         raise NotImplementedError(f'A file with "{field}" is not implemented')
 
-    context.applicable = getattr(context, 'applicable', True) and applicable
```

Comment:
Why was this removed. This is necessary for "chaining" / "combining" multiple givens from this pattern.

Reply:
That was unintentional. I've commented some things out (including the mentioned line), worked on it on another branch to test it with the rules I wrote and copy-pasted it back. I see now how that can lead to errors, it would then be a better/more careful practice to actually develop on the branch that will be pushed ..